index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jim Ntare | Software Engineer</title>
+    <link rel="stylesheet" href="./build/styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="nav_bar flex">
+        <div class="logo">My Logo</div>
+        <div class="nav_items flex">
+          <div class="nav_item">Portfolio</div>
+          <div class="nav_item">About</div>
+          <div class="nav_item">Contact</div>
+        </div>
+        <div class="hamburger" onclick="toggle(this)">
+          <div class="burger-one"></div>
+          <div class="burger-two"></div>
+          <div class="burger-three"></div>
+        </div>
+      </div>
+    </header>
+    <div class="container flex-col">
+      <div class="shapes flex">
+        <div class="sh-group sh-group-one">
+          <div class="rectangle-left"></div>
+          <div class="semi-circle-left"></div>
+        </div>
+        <div class="sh-group sh-group-two">
+          <div class="circle-mid-trans"></div>
+        </div>
+        <div class="sh-group sh-group-three">
+          <div class="rectangle-mid"></div>
+          <div class="circle-mid-opaque"></div>
+        </div>
+        <div class="sh-group sh-group-five">
+          <svg class="sh-circle-top" width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <circle r="19" transform="matrix(1 0 0 -1 19.5 19.5)" fill="white" stroke="#EBEBFF"/>
+            </svg>            
+        </div>
+        <div class="sh-group sh-group-four flex">
+          <div class="rectangle-right"></div>
+          <svg
+            class="semi-ring-right"
+            viewBox="0 0 210 213"
+            fill="#6070FF"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M209.581 57.0278C200.949 41.7052 188.83 28.6308 174.205 18.863C159.581 9.09529 142.861 2.90884 125.401 0.804592C107.94 -1.29966 90.2299 0.737445 73.7033 6.75095C57.1767 12.7645 42.2984 22.5853 30.2733 35.4184C18.2481 48.2515 9.41404 63.736 4.48635 80.6183C-0.441345 97.5005 -1.32415 115.306 1.90941 132.593C5.14297 149.88 12.402 166.162 23.0988 180.122C33.7955 194.081 47.6293 205.326 63.4801 212.945L84.3582 169.508C75.3281 165.168 67.447 158.762 61.3531 150.809C55.2591 142.856 51.1237 133.58 49.2815 123.732C47.4394 113.883 47.9423 103.74 50.7496 94.1219C53.5569 84.5041 58.5896 75.6825 65.4404 68.3716C72.2911 61.0606 80.7672 55.4656 90.1824 52.0397C99.5976 48.6138 109.687 47.4533 119.634 48.6521C129.582 49.8509 139.107 53.3753 147.438 58.94C155.77 64.5046 162.674 71.9532 167.592 80.6824L209.581 57.0278Z"
+              fill="#6070FF"
+            />
+          </svg>
+        </div>
+      </div>
+      <div class="text-container">
```

Comment:
- [ ] [OPTIONAL] I highly recommend you make this change. I think it's better to use the HTML semantic `section` tag instead of a div.